sdk/servicebus/azure-servicebus/api_review.py

```diff
@@ -0,0 +1,588 @@
+
+    def __enter__(self):
+    def __exit__(self):
+
+    def close(self) -> None:
+
+    def get_properties(self) -> Dict[str, Any]:
+
+    def list_session_ids(self) -> List[str]:
+
+    def send(
+        self,
+        messages,
+        session : str = None,
+        message_timeout : float = None
+    ) -> None:
+    def schedule(
+        self,
+        messages,
+        schedule_time : datetime,
+        session : str = None
+    ) -> List[int]:
+    def cancel_scheduled_messages(self, sequence_numbers : List[int]) -> None:
+
+
+class ServiceBusReceiverClient:
+    def __init__(
+        self,
+        fully_qualified_namespace : str,
+        entity_name : str,
+        credential: TokenCredential,
+        subscription_name : str = None,
+        session : str = None,
+        mode : ReceiveSettleMode = ReceiveSettleMode.PeekLock
+        logging_enable: bool = False,
+        http_proxy: dict = None,
+        transport_type: TransportType = None,
+        retry_total : int = 3,
+    ) -> None:
+    @classmethod
+    def from_queue(
+        cls,
+        fully_qualified_namespace : str,
+        queue_name : str,
+        credential : TokenCredential,
+        session: str = None,
+        mode : ReceiveSettleMode = ReceiveSettleMode.PeekLock
+        **kwargs
+    ) -> ServiceBusReceiverClient:
+    @classmethod
+    def from_topic_subscription(
+        cls,
+        fully_qualified_namespace : str,
+        topic_name : str,
+        subscription_name : str,
+        credential: TokenCredential,
+        session: str = None,
+        mode : ReceiveSettleMode = ReceiveSettleMode.PeekLock
+        **kwargs
+    ) -> ServiceBusReceiverClient:
+    @classmethod
+    def from_connection_string(
+        cls,
+        conn_str : str,
+        entity_name : str = None,
+        subscription_name : str = None,
+        session: str = None,
+        mode : ReceiveSettleMode = ReceiveSettleMode.PeekLock
+        **kwargs
+    )-> ServiceBusReceiverClient:
+
+    def __enter__(self):
+    def __exit__(self):
+
+    def close(self) -> None:
+
+    def get_properties(self) -> Dict[str, any]:
+
+    def __iter__(self):
+    def __next__(self):
+    def next(self):
+
+    def peek(
+        self,
+        count : int = 1,
+        start_from : int = None
+    ) -> List[PeekMessage]:
+    def receive_deferred_messages(
+        self,
+        sequence_numbers : List[int],
+    ) -> List[DeferredMessage]:
+    def receive(
+        self,
+        max_batch_size : int = None,
+        timeout : float = None,
+    ) -> List[Message]:  # Pull mode receive
+    def settle_deferred_messages(
+        self,
+        settlement : str,  # TODO: In T1 settlement is just string like 'completed', 'suspended', 'abandoned', can we improve this parameter to be Enum or remove this method?
+        messages : List[DeferredMessage],
+        **kwargs
+    ) -> None:  # Batch settle deferred messages
+
+    def list_sessions(self) -> List[str]:
```

Comment:
If we're returning a list of strings here, we should align naming with the sender: `list_session_ids`